app/spell_check.py

```diff
@@ -9,6 +10,17 @@ def parse_txt(raw_input):
             word_list.append(word)
     return word_list
 
+def parse_txt_other_lang(raw_input):
+    init_word_list = re.split(' ', raw_input)
+    word_list = []
+    for word in init_word_list:
+        if word and word != ' ':
```

Comment:
To follow the refactoring that we are in the midst of with the english parser, we should be keeping track of all white space and not dropping the spaces.

Reply:
I agree, I had built this prior to that decision and forgot to go back and remedy it, I'll make another commit to remedy that

Reply:
This was removed in one of the PRs that has now hit development

Reply:
Regex PR